File: clients/dotnet/enums/TaxType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Avalara.AvaTax.RestClient
{
    /// <summary>
    /// 
    /// </summary>
    public enum TaxType
    {
        /// <summary>
        /// 
        /// </summary>
        ConsumerUse,

        /// <summary>
        /// 
        /// </summary>
        Excise,

        /// <summary>
        /// 
        /// </summary>
        Fee,

        /// <summary>
        /// 
        /// </summary>
        Input,

        /// <summary>
        /// 
        /// </summary>
        Nonrecoverable,

        /// <summary>
        /// 
        /// </summary>
        Output,

        /// <summary>
        /// 
        /// </summary>
        Rental,

        /// <summary>
        /// 
        /// </summary>
        Sales,

        /// <summary>
        /// 
        /// </summary>
        Use,


    }
}

File: clients/php/enums/TaxType.php
<?php
/*
 * AvaTax Enum Class
 *
 * (c) 2004-2016 Avalara, Inc.
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
namespace Avalara.AvaTax;

/**
 * @author Ted Spence <ted.spence@avalara.com>
 * @author Bob Maidens <bob.maidens@avalara.com>
 */class TaxType extends AvaTaxEnum 
{

    const ConsumerUse = "ConsumerUse";
    const Excise = "Excise";
    const Fee = "Fee";
    const Input = "Input";
    const Nonrecoverable = "Nonrecoverable";
    const Output = "Output";
    const Rental = "Rental";
    const Sales = "Sales";
    const Use = "Use";
}
